Pusher.cpp

```diff
@@ -30,8 +30,9 @@
 #include <string.h>
 #include <unistd.h>
 
-Pusher::Pusher(string cerFileName) {
+Pusher::Pusher(string cerFileName, string passWord) {
```

Comment:
Yes, you are right, password is an one word. I always write passWord, this unprofessional habit has started from my early programming. Sometimes I write `int mian(){}`, then I have no idea what is going on with the complain of compiler, besides I barely undertand the debug info from compiler. It was a painful growth for the foreign programmer who has an unsoild English reading and writing skills. I think my English skills better than before, because I always meet people like you, give me a polite correction, then I feel so encouraging to commite my midifications.